internal/deviations/deviations.go

```diff
@@ -69,6 +69,9 @@ import "flag"
 var (
 	InterfaceEnabled = flag.Bool("deviation_interface_enabled", false,
 		"Device requires interface enabled leaf booleans to be explicitly set to true.  Full OpenConfig compliant devices should pass both with and without this deviation.")
+	
```

Comment:
gofmt (CI checks) don't like the stray characters on this line

Reply:
thanks!, fixed the issue.